content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -307,12 +307,14 @@ capabilities such as manipulating the network stack or accessing hardware device
 
 {{< feature-state for_k8s_version="v1.26" state="stable" >}}
 
-In Windows, you can create a [Windows HostProcess pod](/docs/tasks/configure-pod-container/create-hostprocess-pod)
-by setting the `windowsOptions.hostProcess` flag on the security context of the pod spec. All containers in these
-pods must run as Windows HostProcess containers. HostProcess pods run directly on the host and can also be used
-to perform administrative tasks as is done with Linux privileged containers. In order to use this feature, the
-`WindowsHostProcessContainers` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) must be enabled.
+This feature cannot be disabled with Kubernetes version 1.26 and later. In order to use this feature with Kubernetes 1.25 
+or earlier, the `WindowsHostProcessContainers` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) 
+must be enabled.
 
+In Windows, you can create a [Windows HostProcess pod](/docs/tasks/configure-pod-container/create-hostprocess-pod) by setting the 
+`windowsOptions.hostProcess` flag on the security context of the pod spec. All containers in these
+pods must run as Windows HostProcess containers. HostProcess pods run directly on the host and can also be used
+to perform administrative tasks as is done with Linux privileged containers. 
 
```

Comment:
Do you need to set the Pod OS for this to work?

Reply:
Pod OS does not need to be set for this to work.

Reply:
Although it might be good add a check during api-server admission to warn / fail if Pod OS is set to Linux and any of the `securityContext.WindowsOptions` fields are set.

Reply:
Actually this it is already the case that if Pod OS is set to `linux` and windowsOptions are set the pod will be rejected.
This is documented at https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec

> If the OS field is set to linux, the following fields must be unset: -securityContext.windowsOptions

Reply:
The link provided has all additional details, I don't think more is required here.